openeo/internal/processes.py

```diff
@@ -0,0 +1,902 @@
+def ln(cube, x):
+    """The natural logarithm is the logarithm to the base *e* of the number `x`. This
+process is an alias for the *log* process with the base set to *e*: `log(x,
```

Comment:
FYI: the `textwrap` module has an `indent` function (https://docs.python.org/3/library/textwrap.html#textwrap.indent)  with which you can make the indentation of these docblocks nicer

Reply:
Thanks! I used it to make the comments prettier.